service/src/main/java/bio/terra/profile/service/azure/AzureService.java

```diff
@@ -54,6 +61,26 @@ public List<AzureManagedAppModel> getAuthorizedManagedAppDeployments(
         .toList();
   }
 
+  /**
+   * Gets the resource providers in a subscription that are in either the "Registered" or
+   * "Registering" state.
```

Comment:
Out of curiosity, what other states could they be in?

Reply:
NotRegistered

Reply:
Ah, I see. I was imagining it only returned things that they had registered at some point, but possibly things could be in an error state.